pkgs/development/tools/continuous-integration/forgejo-actions-runner/default.nix

```diff
@@ -7,17 +7,17 @@
 
 buildGoModule rec {
   pname = "forgejo-actions-runner";
-  version = "3.0.1";
+  version = "3.1.0";
 
   src = fetchFromGitea {
     domain = "codeberg.org";
```

Comment:
The upstream repo is now https://code.forgejo.org/forgejo/runner (keeping the read-only mirror would be fine for me though).